scalding-core/src/main/scala/com/twitter/scalding/examples/WordCountJob.scala

```diff
@@ -3,8 +3,10 @@ package com.twitter.scalding.examples
 import com.twitter.scalding._
 
 class WordCountJob(args: Args) extends Job(args) {
-  TextLine(args("input")).read.
-    flatMap('line -> 'word) { line: String => line.split("\\s+") }.
-    groupBy('word) { _.size }.
-    write(Tsv(args("output")))
+  TypedPipe.from(TextLine(args("input")))
+    .flatMap { line  => line.split("\\s+") }
```

Comment:
could add:
`// the compiler knows line is a string, so we can call split without specifying a type unlike in the fields API`